src/test/ui/enum/union-in-enum.rs

```diff
@@ -0,0 +1,11 @@
+//This test checks that the union keyword
```

Comment:
Nit: A space after `//` would make the paragraph more readable.

Reply:
done